question-servers/freeformPythonLib/prairielearn.py

```diff
@@ -128,11 +128,33 @@ def inner_html(element):
     return html
 
 
+def compat_has(object, attrib):
+    if attrib in object:
+        return True
+    new_attrib = attrib.replace('-', '_')
+    return new_attrib in object
+
+
+def compat_get(object, attrib, default):
+    if attrib in object:
+        return object[attrib]
+    new_attrib = attrib.replace('-', '_')
+    return new_attrib in object
+
+
+def compat_array(arr):
+    new_arr = []
+    for i in arr:
+        new_arr.append(i)
+        new_arr.append(i.replace('-', '_'))
+    return new_arr
+
+
 def check_attribs(element, required_attribs, optional_attribs):
     for name in required_attribs:
-        if name not in element.attrib:
+        if not compat_has(element.attrib, name):
```

Comment:
Why do we call `compat_has()` here rather than the existing `has_attrib()`? It looks to me like they do the same thing, but I'm probably missing something.

Reply:
Nope, good catch! Fixed.